src/ngraph/runtime/cpu/pass/cpu_layout.cpp

```diff
@@ -474,6 +475,26 @@ namespace ngraph
                     }
                 }
 
+                template <>
+                void CPULayout::LAYOUT_DECL(ngraph::op::ConvolutionAdd)
+                {
+                    if (mkldnn_utils::use_mkldnn_kernel(node.get()))
+                    {
+                        vector<memory::desc> i_mds;
+                        vector<memory::desc> o_mds;
+                        ConvolutionLayout<ngraph::op::ConvolutionAdd, false, false>(
+                            node, i_mds, o_mds);
+                        // Force second input to sum to use the same layout as convolution output
+                        i_mds.push_back(o_mds[0]);
+                        node = insert_input_conversions(external_function, node, i_mds);
+                        set_output_layouts(node, o_mds);
+                    }
+                    else
+                    {
+                        set_native_layouts(external_function, node);
```

Comment:
might be better to throw an error if not mkldnn here as well.

Reply:
You mean instead of using set_native_layouts() , throw ngraph_error() ? 
I will do it, but i don't understand why though.

Reply:
There is no non-mkldnn kernel to execute an in-place ConvolutionAdd. So we shouldn't hit this case ever.

Reply:
Ok. Done